modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/rendering/FragmentRenderer.java

```diff
@@ -32,14 +37,18 @@ public final class FragmentRenderer
 
     private final ContentService contentService;
 
+    private final LayoutDescriptorService layoutDescriptorService;
+
     private final RendererDelegate rendererDelegate;
 
     private final FragmentPageResolver fragmentPageResolver = new FragmentPageResolver();
 
-    public FragmentRenderer( final ContentService contentService, final RendererDelegate rendererDelegate )
+    public FragmentRenderer( final ContentService contentService, final LayoutDescriptorService layoutDescriptorService,
```

Comment:
Why LayoutDscriptorService is needed, but PartDescriptorService is not? it is needed in other paces